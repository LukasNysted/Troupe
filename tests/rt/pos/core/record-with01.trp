let val point0 = { x = 0, y = 0 } 
    val point1 = { point0 with x = 10 }
in point1 
end