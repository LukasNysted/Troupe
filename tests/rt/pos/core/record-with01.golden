2020-10-16T12:42:55.597Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: {x=10@{}%{}, y=0@{}%{}}@{}%{}
